site/content/docs/4.3/examples/carousel/carousel.css

```diff
@@ -38,11 +38,6 @@ body {
 /* MARKETING CONTENT
 -------------------------------------------------- */
 
-/* Center align the text within the three columns below the carousel */
-.marketing .col-lg-4 {
-  margin-bottom: 1.5rem;
-  text-align: center;
```

Comment:
If remove this `text-align: center;`, please remove the below too:
```scss
.marketing .col-lg-4 p {
  margin-right: .75rem;
  margin-left: .75rem;
}
```

Reply:
Reverted this change